Додає розмітку контактів

biochipman opened the pull request:
- Іконки та контентне зображення карти будуть додані як тільки буде доступний спрайт та images відповідно

biochipman committed:
Додає розмітку контактів

- Іконки та контентне зображення карти будуть додані як тільки буде доступний спрайт та images відповідно

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -1 +1,42 @@
-<section></section>
+<section class="contacts">
+  <h1 class="visually-hidden">Контакти Міміно</h1>
```
Тут буде прихований заголовок рівня h2. Текст заголовка -  Контакти. Бо на сторінці може бути лише один заголовок рівня h1

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -1 +1,42 @@
-<section></section>
+<section class="contacts">
+  <h1 class="visually-hidden">Контакти Міміно</h1>
+  <h2 class="section-logo">Mimino</h2>
```
це буде просто абзац тексту

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -1 +1,42 @@
+<section class="contacts">
+  <h1 class="visually-hidden">Контакти Міміно</h1>
+  <h2 class="section-logo">Mimino</h2>
+  <p>Ресторанно-готельний комплекс</p>
```
Це можна зробити заголовком третього рівня

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -1 +1,42 @@
+  <h1 class="visually-hidden">Контакти Міміно</h1>
+  <h2 class="section-logo">Mimino</h2>
+  <p>Ресторанно-готельний комплекс</p>
+  <address>
```
Додай клас mimino-address. Навіть якщо ти його не стилізуєш, він приймає участь у назві класів дітей і нащадків

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -1 +1,42 @@
+  <address>
+    <ul class="mimino-address__list list">
+      <li class="mimino-address__item">
+        <p class="mimino-address__title">Ресторан</p>
```
Заголовок 4-го рівня

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -1 +1,42 @@
+        <svg>
+          <use></use>
+        </svg>
+        <a href="tel:+380955559533" class="mimino-address__link"
```
Додай клас link для скидання стандартних стилів

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -1 +1,42 @@
+        >
+      </li>
+      <li class="mimino-address__item">
+        <p class="mimino-address__title">Готель</p>
```
Заголовок четвертого рівня

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -1 +1,42 @@
+        <svg>
+          <use></use>
+        </svg>
+        <a href="tel:+380668491089" class="mimino-address__link"
```
Додай клас link для скидання стандартних стилів

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -1 +1,42 @@
+        <a href="tel:+380668491089" class="mimino-address__link"
+          >066 849 10 89</a
+        >
+        <a href="tel:+380672758540" class="mimino-address__link"
```
Додай клас link для скидання стандартних стилів

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -1 +1,42 @@
+          <use></use>
+        </svg>
+        <a
+          class="mimino-address__link"
```
Додай клас link для скидання стандартних стилів

kuhai-anna reviewed (changes requested):
Гарна робота! Чекаю на внесення змін

biochipman committed:
Вносить правки в секцію контактів

- Вносить правки для покращення семантичності коду
- Додає розмітку зображення мапи

kuhai-anna reviewed (approved):
Чудово!
